docs/benchmarks.rst

```diff
@@ -1,25 +1,4 @@
 Benchmarks
 ==========
```

Comment:
We should replace internal links to this document with a direct link to https://github.com/ivirshup/anndata-benchmarks.

To leave this document in to not break external links coming in, we can do this:

```suggestion
:orphan:

Benchmarks
==========
```

Reply:
Alex and I talked about this a bit.

The ASV benchmarks are really more of a development tool than something that's useful to show to users. However, it would be good to have something showing that AnnData is efficient in the docs.

Also, maybe the benchmarking code should live in this repo. It's what other projects do.

Reply:
Notebook is coming!